docs/sql/mysql/create_push_server_schema.sql

```diff
@@ -87,5 +92,7 @@ CREATE INDEX `push_campaign_user_campaign` ON `push_campaign_user`(`campaign_id`
 CREATE INDEX `push_campaign_user_detail` ON `push_campaign_user`(`user_id`);
 
 CREATE INDEX `push_inbox_id` ON `push_inbox` (`inbox_id`);
-CREATE INDEX `push_inbox_user_app` ON `push_inbox` (`user_id`, `app_id`);
+CREATE INDEX `push_inbox_user_app` ON `push_inbox` (`user_id`);
 CREATE INDEX `push_inbox_user_read` ON `push_inbox` (`user_id`, `read`);
+
+CREATE INDEX `push_inbox_app_inbox` ON `push_inbox_app` (`inbox_id`, `app_credentials_id`);
```

Comment:
This index seems a bit strange, it should be unique, we probably do not want to allow duplicates. Probably a good candidate for a primary key.

Reply:
Absolutely, I will fix this.